lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Hmm, not if the presence validator is automatically executed for all parameters listed under default_settings. By the looks of it, this is supposed to be an optional parameter to begin with, so removing it from default_settings looks ok to me.